src/pygeoprocessing/geoprocessing.py

```diff
@@ -2540,7 +2541,6 @@ def _fill_work_queue():
     # indexes
     LOGGER.debug('start worker thread')
     write_queue = queue.Queue(10)
-    worker_list = []
```

Comment:
This was left over from the last changes and not removed I take it?

Reply:
Yes!